mappings/net/minecraft/component/type/AttributeModifiersComponent.mapping

```diff
@@ -23,6 +28,37 @@ CLASS net/minecraft/unmapped/C_azfkqhlm net/minecraft/component/type/AttributeMo
 			ARG 1 attribute
 		METHOD m_vdrgfbcy (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/kinds/App;
 			ARG 0 instance
+	CLASS C_kqgfvdat Display
+		METHOD m_brnkyddm getDefault ()Lnet/minecraft/unmapped/C_azfkqhlm$C_kqgfvdat;
+		METHOD m_eakhcuqk appendTooltip (Ljava/util/function/Consumer;Lnet/minecraft/unmapped/C_jzrpycqo;Lnet/minecraft/unmapped/C_cjzoxshv;Lnet/minecraft/unmapped/C_hdbqsqsm;)V
+			ARG 1 append
+			ARG 3 attribute
+			ARG 4 modifier
+		METHOD m_oopxewmk (Lnet/minecraft/unmapped/C_azfkqhlm$C_kqgfvdat$C_ojlwbbdy;)Lcom/mojang/serialization/MapCodec;
+			ARG 0 type
+		METHOD m_pfcndfzi createOverride (Lnet/minecraft/unmapped/C_rdaqiwdt;)Lnet/minecraft/unmapped/C_azfkqhlm$C_kqgfvdat;
+		METHOD m_tfwgfjdw getType ()Lnet/minecraft/unmapped/C_azfkqhlm$C_kqgfvdat$C_ojlwbbdy;
+		METHOD m_vdlizjqr getHidden ()Lnet/minecraft/unmapped/C_azfkqhlm$C_kqgfvdat;
+		CLASS C_dzmzhicx Default
+			FIELD f_bnylsies INSTANCE Lnet/minecraft/unmapped/C_azfkqhlm$C_kqgfvdat$C_dzmzhicx;
+		CLASS C_mgthfrsm TextOverride
```

Comment:
I initially named this `Override`, but after I rebased to 25w18a, that started causing errors for javadoc:
> error: incompatible types: Override cannot be converted to Annotation

It was getting it confused with `java.lang.Override`.

If there's a way we can ensure the fake source gets generated with qualified `java.lang.Override`s in cases like this that would be ideal.